### ISSUE TITLE
one dimension data

### ISSUE THREAD

**Issue by wxw-cyber** [shkarupa-alex/tfvan#1]
Hello, how do I use `tfvan` if my data is one dimensional?

**Comment by shkarupa-alex** [comment]
Hello, in the same way as any other backbone in keras_application.
There are two options:
1) You can repeat your data 3 times across channel dimension and then concat. (I think this is the easiest option)
2) You can transfer pretrained weights to empty model with 1-channel input:
- create 2 models (1 with pretrained weights and 3 channels, 1 with random init and 1 channel)
- find weights in these two models with different shape (first conv weighs)
- sum that weight accross channel dim
- use this summed weight and all other weights from pretrained model and set them to model with 1 weight: model.set_weights([..])


P.S.
Try to search in google how to do that for some backbone like keras resnet50. For tfvan alghoritm will be the same.
